docs/src/main/resources/templates/scaling.html

```diff
@@ -0,0 +1,115 @@
+<h4 id="scaling_intro">
+  Introduction
+  <a class="headerlink" href="#scaling_intro" title="Permalink to this headline">¶</a>
+</h4>
+<p>
+  The print has a state do store the job queue and the job state, then we should store this start to be able
+  to do Horizontal scaling.
+</p>
+<h4 id="scaling_solus">
+  Solution
+  <a class="headerlink" href="#scaling_solus" title="Permalink to this headline">¶</a>
+</h4>
+<p>
+  To store this state we use a ProtgreSQL database, she should be configured with the following Java system
```

Comment:
this -> the 
(or be more specific what state exactly is being stored)

ProtgreSQL -> PostgreSQL

she -> what does she refer to? I assume you mean the database connection

Java system -> why Java system properties? These are simply database connection properties, not necessarily system properties?

Reply:
The Java -D parameter is documented as system property...
```
    -D<name>=<value>
                  set a system property
```

Reply:
OK,
but why "the database should be configured"? I think you mean "the database connection should be configured"?